Makefile

```diff
@@ -170,6 +170,12 @@ BUILD_DATE      ?= $$(git log -1 --format="%at" | xargs -I{} date -d @{} +%Y-%m-
 get_version:
 	@echo -n v$(VERSION)
 
+fmt: ## Run go fmt against code.
+	go fmt ./...
+
+vet: ## Run go vet against code.
+	go vet ./..
+
```

Comment:
I used to remove these targets mostly because we're relying on `golangci-lint` which offers `fmt` (as `gofmt`) and `vet` (as `govet`) too.

We could reuse the `golint` target before building the binary, rather.

Reply:
Thank you for your feedback. I've reviewed the changes made to the Makefile, and they indeed align with your suggestions. Here's a summary of the modifications:

The 'fmt' and 'vet' targets have been removed from the Makefile.
The 'build' target now depends on 'generate' and 'golint' instead of 'generate', 'fmt', and 'vet'.
The 'run' target no longer depends on 'fmt' and 'vet'.